builtin/providers/aws/resource_aws_iam_policy_attachment_test.go

```diff
@@ -303,9 +307,13 @@ resource "aws_iam_policy_attachment" "test-attach" {
 }
 
 const testAccAWSPolicyPaginatedAttachConfig = `
+resource "random_id" "user_id" {
+  byte_length = 10
+}
+
 resource "aws_iam_user" "user" {
     count = 101
-    name = "${format("paged-test-user-%d", count.index + 1)}"
+    name = "${format("paged-test-user-${random_id.user_id.hex}-%d", count.index + 1)}"
```

Comment:
I was trying to figure this out and what you said makes total sense. I'll make the changes